spring-cloud-gcp-samples/spring-cloud-gcp-data-multi-sample/README.adoc

```diff
@@ -0,0 +1,23 @@
+= Spring Cloud GCP Cloud Cloud Spanner and Datastore Combined Usage Example
```

Comment:
I'm open to just calling this sample a kitchen-sink sample and not describing it too much in the readme.
We might want to use it for other combinations as well like Logging and Trace which have been problematic in the past too.

Reply:
Any suggestions on the name ?

Reply:
I thought a little more and I want to keep this one just about multi-spring-data usage. I think that concept alone needs a clean slate to display, and if we add other junk in here it might be hard to read.

Reply:
I guess where I'm coming from is that I feel like this is not a very common use-case that needs a sample. The primary reason that we have this sample is to have an integration test. If that's not the case, and we really think it's a useful example for learning how to use the framework, let's leave it this way.

Reply:
For the name, I would literally call it `spring-cloud-gcp-kitchen-sink-sample`.

Reply:
Hmm yea actually Ray talked to me in person about this a while ago. I expressed some surprise he wanted to demo this combined usage on stage and he said it was quite a draw actually. I think if he's using it on stage we ought to have a sample to back it up too.

Reply:
I would be curious to see his demo. The one I saw he was just switching examples from Datastore to Spanner. Maybe he was forgetting to remove the dependency. @saturnism